src/projects/intelligent-user-interfaces-for-virtual-healthcare.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: layouts/base.njk
+title: "Intelligent User Interfaces for Virtual Healthcare"
+description: >-
+  *Intelligent User Interfaces for Virtual Healthcare* aims to reduce barriers to accessing healthcare through inclusive guidelines for (virtual or in-person) healthcare and an approach to personalization in these settings.

```

Comment:
The emphasis on the project title in the description is inconsistent across projects.

Reply:
Actually, the formatting was inconsistent across multiple projects - some using bold, some using italics. I've just removed all that formatting for now.